velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -1125,6 +1125,28 @@ struct ArrayUnionFunction {
   }
 };
 
+/// This class implements the array_compact function.
+///
+/// DEFINITION:
+/// array_compact(x) -> array
+/// Remove all null elements from array x.
+template <typename T>
+struct ArrayCompactFunction {
```

Comment:
Shouldn't the function be put under sparksql folder?

Reply:
ArrayRemovedNullFunction reused. And I see other reused-function still under presto foler, better to keep the status?

Reply:
@dcoliversun, let's keep other functions unchanged for some time. We should confirm there is no corner case that Presto and Spark produce different result before doing the refactor.